app/test_main.py

```diff
@@ -1,3 +1,48 @@
+import pytest
+
 from app.main import get_human_age
```

Comment:
Make sure you've tests checks all edged situations such as:

if function receive data out of normal range, such as negative numbers, zero, or realy large numbers;
if the function receives an incorrect type of data it raises the correct exception.